src/h5web/visualizations/nexus/hooks.ts

```diff
@@ -73,15 +54,16 @@ export function useNxData(group: HDF5Group, metadata: HDF5Metadata): NxData {
     };
   });
 
-  const errorsLink =
-    getLink(`${signalName}_errors`, group) || getLink('errors', group);
+  const errorsDataset =
+    getChildEntity(group, `${signalName}_errors`) ||
+    getChildEntity(group, 'errors');
   const errorsValue =
-    errorsLink && isReachable(errorsLink) && values[errorsLink.id];
+    errorsDataset && isMyDataset(errorsDataset) && values[errorsDataset.id];
```

Comment:
Pretty much equivalent for now but there's probably improvement to be made.

Reply:
Should be fine for now. If we attach `id` to `MyHDF5Entity`, `errorsDataset && values[errorsDataset.id]` will be sufficient ?